templates/customize-grid/grid-levels.html

```diff
@@ -0,0 +1,47 @@
+<div id="level">
+  <table id="grid-table">
+    <thead>
+      <tr>
+        <th>Students</th>
+        {% for adventure in class_adventures[level] %}                   
+          <th>{{ adventure }}</th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody>
+      {% for row in matrix_values %}
+        <tr>
+          {% set student_index = loop.index0 %}
+          {% set student = class_info['students'][student_index] %}
+          <th>{{ student }}</th>
+          {% for checkbox in row %}
+            {% set adventure_index = loop.index0 %}
+            {% set adventure_id = adventure_names[class_adventures[level][adventure_index]] %}
+            {% set student_adventure_id = student ~ '-' ~ adventure_id ~ '-' ~ level %}
+            {% if checkbox == 0 %}
```

Comment:
I think you can save a lot of code here by joining the conditions `checkbox == 0` and `checkbox == 1` and checking and and decide to check or not inside the html element with something like


```
<input type="checkbox" name="myCheckbox" class="checkbox-wrapper" {% if checkbox == 1%}value="checked"{% endif %} [...]>
```